### ISSUE TITLE
Continuation of Roadmap & Discussion

### ISSUE THREAD

**Issue by tannerflick4** [Altemio/SevenKingdoms-master#45]
So, the old issue thread is 285 comments long, and therefore my internet hates to load it. I prefer if we start this thread new, it doesn't need to be fancy intro, just making it easier to load :)

**Comment by RowanMaBoot** [comment]
Yeah, the old thread was getting hectic! Especially the email chain..

I'm going to see about changing our workflow on the Git, and will try to split off issues in the future to separate issues, which can then be linked to the roadmap for releases (so we can keep track of what needs to be fixed for the next release).
